frontend/src/metabase/visualizations/components/ChartCaption.jsx

```diff
@@ -18,7 +18,8 @@ const ChartCaption = ({
   actionButtons,
   onChangeCardAndRun,
 }) => {
-  const title = settings["card.title"] || series[0].card.name;
+  const cardTitle = settings["card.title"];
+  const title = typeof cardTitle === "string" ? cardTitle : series[0].card.name;
```

Comment:
The fix: distinguish between `null` (or other falsy) and an empty string.

Reply:
nit: use `??` instead